google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingEventEnhancer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
We have changed the license header to `Copyright YYYY Google LLC`. Could you update this. Copying from `LoggingAppender.java` should work fine.